pkg/controller/controller_manger_test.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+import (
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = Describe("controller", func() {
+	BeforeEach(func() {
+	})
+
+	AfterEach(func() {
+	})
+
+	Describe("", func() {
+
+		It("should do something", func() {
+
+		})
+	})
+})
```

Comment:
nit: filename has "manger" --> manager :)

Reply:
Should `Describe` be "controller manager", and `//TODO` comments be needed ?